x-pack/filebeat/module/crowdstrike/falcon/test/falcon-events.log-expected.json

```diff
@@ -0,0 +1,572 @@
+[
+    {
+        "@timestamp": "2020-02-27T19:12:14.000Z",
+        "service.type": "crowdstrike",
+        "input.type": "log",
+        "crowdstrike.metadata.customerIDString": "8f69fe9e-b995-4204-95ad-44f9bcf75b6b",
+        "crowdstrike.metadata.offset": 1045,
+        "crowdstrike.metadata.eventType": "RemoteResponseSessionStartEvent",
+        "crowdstrike.metadata.eventCreationTime": 1582830734000,
+        "crowdstrike.metadata.version": "1.0",
+        "crowdstrike.event.SessionId": "6020260b-0398-4d41-999d-5531b55522de",
+        "crowdstrike.event.HostnameField": "hostnameofmachine",
+        "crowdstrike.event.UserName": "first.last@company.com",
+        "crowdstrike.event.StartTimestamp": 1582830734,
+        "event.module": "crowdstrike",
+        "event.dataset": "crowdstrike.falcon",
+        "event.kind": "event",
+        "event.action": "remote_response_session_start_event",
+        "event.type": "info",
+        "host.name": "hostnameofmachine",
+        "user.name": "first.last@company.com",
```

Comment:
this looks like user.email, will these always be emails?  maybe switch on the presence of@ for email vs name?

Reply:
This is an interesting question that I don't know the answer to since we don't have access to the Falcon platform. This is the username of the Crowdstrike Falcon user that is initiating a remote session to the endpoint. I'm not sure if it's guaranteed to be an email. But it's definitely meant to be the username since the original field is UserName (crowdstrike.event.UserName).

Reply:
Yeah it's common to see emails in "user id" or "user name" fields in different systems.

Similar to the point I made about `UserId` vs `UserName` in a comment on `AuthActivityAuditEvent`, I'd suggest filling our `user.name` with this email (as you did here), to try to follow their semantics as closely as possible.

Then perhaps do a check and also fill the `user.email` field with the same value if the value in question looks like an email. I wouldn't overthink the check, the presence of `@` in the field is probably enough.